utils/mappings.js

```diff
@@ -318,12 +317,6 @@ export function reIndex({
 		body.mappings = { properties: { ...properties }, ...rest };
 	}
 
-	if (enableNgram) {
```

Comment:
@jyash97 Why are we removing this code?

Reply:
I am handling this now on Mappings state directly. So from now on, reindex will contain the latest mappings based on search relevance.